doc/man7/proxy-certificates.pod

```diff
@@ -0,0 +1,359 @@
+=pod
+
+=head1 NAME
+
+proxy-certificates - Proxy certificates in OpenSSL
+
+=head1 DESCRIPTION
+
+Proxy certificates are defined in
+L<RFC 3820|https://tools.ietf.org/html/rfc3820>.  They are used to
```

Comment:
I believe all other RFC references just have the link in the SEE ALSO section.

Reply:
Ah, thanks.  I intended to do exactly that, but forgot.

Reply:
Oh and also, the official RFC link is https://www.rfc-editor.org/rfc/rfcXXXX.html

Reply:
Actually, this manpage is almost pioneering that.  The only other that I could find was the tsget manual.

Reply:
One of my pending PR's fixes tsget :)